src/renderer/components/watch-video-info/watch-video-info.scss

```diff
@@ -47,38 +49,30 @@
     margin-block-start: 6px;
     padding: 6px;
   }
-}
 
-.viewCount,
-.datePublished {
-  color: var(--secondary-text-color);
-  font-size: 15px;
-  text-align: end;
-
-  @media screen and (max-width: 680px) {
-    text-align: start;
+  .videoOptions {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-end;
+    gap: 4px;
+
+    @media screen and (max-width: 680px) {
+      :deep(.iconDropdown) {
+        inset-inline-start: calc(50% - 20px);
+        inset-inline-end: auto;
+      }
+    }
   }
 }
 
-.viewCount {
-  margin-block: 18px 0;
-  margin-inline: 0;
-}
-
-.datePublished {
-  margin-block: 4px 0;
-  margin-inline: 0;
-
-  @media screen and (max-width: 680px) {
-    margin-block-start: 16px;
-  }
-}
+.videoMetrics {
+  font-size: 12px;
```

Comment:
Please choose a larger font size, 16px is considered the minimum that is readable, especially as the items, that are now on that row, used to be 16px and 15px before (stylelint used to complain about that but afaik you disabled everything, including that check, when you did you block and inline size rules).

Reply:
> 16px is considered the minimum that is readable

Is it okay if I ask what your source is for this claim? The closest I can find is [this article](https://fontsaga.com/why-does-font-size-matter-in-design/) recommending 16 pixels as optimal for mobile, which ends up being fine here as well, as device manufacturers use the reference px size to set the actual rasterizing size based on intended distance of viewing. Not to uphold YouTube as a superior example of things, but they themselves use 12px font for the "1.46M subscribers" text in the image below. For the # of views and likes, they use 14px font. From my research, [12px seems to be the consistent minimum recommended font size](https://accessibility.psu.edu/fontsizehtml/) (16px should generally be used as the minimum for body text, and 12px is acceptable for supplementary information). If we had a stylelint rule that disallowed font sizes below 16px (which seemingly wasn't being enforced, as I see dozens of instances of 12px and even 11px font-sizes in our codebase), I personally think that's unnecessarily stringent. It's useful in places like here for demarcating levels of importance of information and guiding the eye to salient focal points.

![Screenshot_20231115_153641](https://github.com/FreeTubeApp/FreeTube/assets/84899178/1e0a8e1d-36ab-4e3e-9220-e46a0f729451)

Reply:
The stylint checks were optional for so long because we had many different violations in the codebase. Anyway here is the markdown document associated with that accessibility rule:

https://github.com/double-great/stylelint-a11y/tree/main/src/rules/font-size-is-readable

Reply:
The articles cited there in the justification for the rule are the MDN article for `font-size` (which doesn't seem to broach the topic of recommended minimum size) and [this article talking about body text](https://marvelapp.com/blog/body-text-small/). When we add this rule back, we should definitely consider adding it as a warning, as there are quite a few exceptions to that rule.